SAPHana/srHook/SAPHanaSrMultiTarget.py

```diff
@@ -39,8 +39,7 @@ def getEpisode():
     episode = "{0}-{1}".format( datetime.now().strftime('%s') , random.randrange(10000,20000))
     return episode
 
-# TODO: need to move logTimestamp() into the class, because it uses self.tracer.info()
-def logTimestamp(episode, outputMessage):
+def logTimestamp(self, method, episode, outputMessage):
```

Comment:
@angelabriel good catch - should we move "logTimestamp()" to be part of the class, so we do not explicitly need to name "self" as parameter? Just want to discuss this. logTimestamp() should be currently useless outside of class-objects, because I guess such calls would then crash when calling self.trace.info().

Reply:
@fmherschel it's fine for me to move the function inside the class because it's truly the right place for it. I simply choose the same way as it was done in `susChkSrv.py`  to have the same solution in both HA/DR provider hooks.

But if we still will use `self.tracer.info` inside of `logTimestamp` to write the exception error into the trace file, we will not get rid of the `self` parameter because otherwise the function `tracer.info` will not be (easy) available.

And - should we change `susChkSrv.py` as well? To have the same in all hooks?

Reply:
@angelabriel Yes you are right. We should also do it in susChkSrv.py. As far as I remember I moved that fuction out, because pylint blocked me with a message that the class would have to much methods ;). But we should switch-off this pylint-check and re-move the function back to be a class method. I will do so in the angi-project for susChkSrv.

Reply:
fix committed, hope it's as expected.